README.md

```diff
@@ -1,46 +1,22 @@
-Installation
-------------
+# Owning a Home
 
-* `npm install -g gulp`
-* `npm install -g bower`
-* `npm install -g jest-cli`
-* `npm install .`
+[![Build Status](https://travis-ci.org/contolini/dash.svg?branch=master)](https://travis-ci.org/contolini/dash) [![Coverage Status](https://coveralls.io/contolini/contolini/dash/badge.svg)](https://coveralls.io/r/contolini/dash)
```

Comment:
These URLs will need to be changed to use the CFPB org, but maybe not until after this PR?